content/en/docs/Release Management/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Release Management
+linkTitle: "Release Management"
+weight: 120
+---
+
+## Release Management Guide
+
+This document approximates timelines and how to maintain release branches.
+
+### The overall scope of the project
+
+Kyverno simplifies Kubernetes configuration management using policies for security and automation. Kyverno makes it easy for Kubernetes administrators to write and manage Kubernetes policies, and for Kubernetes users (ex., developers) to consume policy results and address issues.
+
+### Release Timelines
+
+Kyverno uses the [Semantic Versioning](https://semver.org/) scheme. Kyverno v1.0.0 was released in Dec 2019. This project follows a given version number MAJOR.MINOR.PATCH.
+
+**MAJOR release:**
```

Comment:
Please use Markdown headings rather than bolding for headings. To do otherwise causes linting and formatting errors, and makes it so the text isn't sometimes portable across different rendering engines.